api/pkg/server/handlers_test.go

```diff
@@ -0,0 +1,58 @@
+package server_test
+
+import (
+	"fmt"
+	"happenedapi/pkg/server"
+	test "happenedapi/pkg/tests"
+	"net/http"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_CreateUploadURLHandler(t *testing.T) {
```

Comment:
Simple test against real s3 bucket. This will likely fail on your environment if you have not run the Terraform and `set_iam_user.sh` script.